lib/yaml_vault/yaml_tree_builder.rb

```diff
@@ -100,6 +104,26 @@ def alias(anchor)
 
     private
 
+    def handle_prefix(value)
+      if @prefix != nil
+        value = "#{@prefix}#{value}"
```

Comment:
In Ruby, String Interpolation converts `nil` to a blank string.
And so, you don't need the guard for `nil`